include/openssl/ui.h

```diff
@@ -266,14 +266,12 @@ DEFINE_STACK_OF(UI_STRING)
  * The different types of strings that are currently supported. This is only
  * needed by method authors.
  */
-enum UI_string_types {
-    UIT_NONE = 0,
-    UIT_PROMPT,                 /* Prompt for a string */
-    UIT_VERIFY,                 /* Prompt for a string and verify */
-    UIT_BOOLEAN,                /* Prompt for a yes/no response */
-    UIT_INFO,                   /* Send info to the user */
-    UIT_ERROR                   /* Send an error message to the user */
-};
```

Comment:
As I said, if necessary I'll remove it.  But if most of the uses are like the one in apps/apps.c (a switch on ```UI_get_string_type```) then *no code changes are necessary* -- apps.c isn't in this PR.